docs/aks-installation.md

```diff
@@ -110,6 +126,103 @@ you should be able to see these Pods running in your cluster:
     pod "tunnelfront-649ff5fb55-5lxg7" deleted
     ```
 
+### Deploying Antrea in `encap` mode
+
+AKS now officially supports [Bring your own Container Network Interface (BYOCNI)](https://learn.microsoft.com/en-us/azure/aks/use-byo-cni).
+Thanks to this, you can deploy Antrea on EKS in `encap` mode, and you will not
+lose access to any functionality. Check the AKS BYOCNI documentation for
```

Comment:
any Antrea functionality?

Reply:
actually I meant it in a more general way, since when deploying Antrea on EKS in encap mode, K8s API aggregation will stop working for example